backend/routes.js

```diff
@@ -213,8 +213,6 @@ const setup = ({ elite2Api, whereaboutsApi, oauthApi, communityApi }) => {
   router.get('/videolink', async (req, res) => {
     res.render('courtsVideolink.njk', {
       user: { displayName: req.session.userDetails.name },
-      homeUrl: '/videolink',
-      title: 'Video link appointment booking',
     })
```

Comment:
Ooh, the homeUrl is used in the header for where clicking the logo will take them to as well as the breadcrumbs. Have we decided we want them to go to DPS if they click the logo?

Reply:
Ahh, my bad.  Thought it was just used in the breadcrumbs. Will change.  Cheers